README.md

```diff
@@ -215,10 +224,10 @@ For better visualization of tabluar results this extension includes a dependency
 
 Notebook Kernels also support other features like [Files in Repo](https://docs.databricks.com/_static/notebooks/files-in-repos.html) to build libraries within your repo, [_sqldf](https://docs.databricks.com/notebooks/notebooks-use.html#explore-sql-cell-results-in-python-notebooks-natively-using-python) to expose results of SQL cells to Python/Pyspark, `%run` to run other notebooks inline with the current notebook and also [dbutils.notebook.run()](https://docs.databricks.com/dev-tools/databricks-utils.html#notebook-utility-dbutilsnotebook).
 
-Whenever a notebook is opened from either the local sync folder or via the [Virtual File System](#file-system-integration) using `dbws:/` URI, the Databricks notebook kernels are the preferred ones and should appear at the top of the list when you select a kernel.
+Whenever a notebook is opened from either the local sync folder or via the [Virtual File System](#file-system-integration) using `wsfs:/` URI, the Databricks notebook kernels are the preferred ones and should appear at the top of the list when you select a kernel.
```

Comment:
I couldn't get notebooks to work. I'm always getting `TypeError: Cannot read properties of undefined (reading 'status')`

![Screenshot 2023-03-16 at 10 20 02](https://user-images.githubusercontent.com/40952/225571704-936bfa1b-4387-411d-bb34-38a6a3040eb4.png)